data-prepper-plugins/log-generator-source/src/test/java/com/amazon/dataprepper/plugins/source/loggenerator/LogGeneratorSourceTest.java

```diff
@@ -31,23 +33,28 @@
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyInt;
 
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import static org.junit.Assert.assertEquals;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
 
+@ExtendWith(MockitoExtension.class)
 public class LogGeneratorSourceTest {
+    @Mock
     private LogGeneratorSourceConfig sourceConfig;
+    @Mock
     private PluginMetrics pluginMetrics;
+    @Mock
     private PluginFactory pluginFactory;
+    @Mock
     private LogGeneratorSource logGeneratorSource;
```

Comment:
You don't want to mock this one since your are testing it

Reply:
Great catch, thanks!